### PR TITLE
ui: allow filtering account by state

### PR DESCRIPTION
**Pull request by rohityadavcloud**
This allows users and admin to filter account by state

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
cc @kishankavala @vishesh92

**Comment by rohityadavcloud** [comment]
@blueorangutan ui

**Review by DaanHoogland — APPROVED** [review]
clgtm

File: ui/src/views/AutogenView.vue [hunk 1]
```diff
@@ -58,7 +58,7 @@
                         ? 'all' : ['publicip'].includes($route.name)
                         ? 'allocated' : ['guestnetwork', 'guestvlans'].includes($route.name)
                         ? 'all' : ['volume'].includes($route.name)
-                        ? 'user' : 'self')"
+                        ? 'user' : '')"
```
[the remark this answers is not in the document]
Comment (rohityadavcloud) [reply ui/src/views/AutogenView.vue:61]: Good catch will check. This PR was for example only.

File: ui/src/views/AutogenView.vue [hunk 2]
```diff
@@ -1580,6 +1581,8 @@ export default {
         } else if (['running', 'stopped'].includes(filter)) {
           query.state = filter
         }
+      } else if (this.$route.name === 'account') {
+        query.state = filter
```
[the remark this answers is not in the document]
Comment (rohityadavcloud) [reply ui/src/views/AutogenView.vue:1585]: Good catch, will review and apply. This PR was only for example impl for Vishesh and Kishan.

**Comment by rohityadavcloud** [comment]
@vishesh92 since you're on this in another PR, could you merge this PR in yours. Thanks.

**Pull request closed without merging**